src/components/overlay/overlay.scss

```
@@ -0,0 +1,193 @@
+@use '../core/styles' as sbb;
+
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin
+@include sbb.host-component-properties;
+
+:host {
+  @include sbb.scrollbar-variables;
+
+  --sbb-overlay-padding-inline: var(--sbb-spacing-fixed-5x);
+  --sbb-overlay-color: var(--sbb-color-black);
+  --sbb-overlay-background-color: var(--sbb-color-milk);
+  --sbb-overlay-width: 100%;
+  --sbb-overlay-height: 100%;
+  --sbb-overlay-inset: 0 auto auto 0;
+  --sbb-overlay-padding-block: var(--sbb-spacing-responsive-s);
```

In the design I see an L.
```suggestion
  --sbb-overlay-padding-block: var(--sbb-spacing-responsive-l);
```